clusters/prod-eu/flux-system/runtime-info.yaml

```diff
@@ -8,7 +8,9 @@ metadata:
   annotations:
     kustomize.toolkit.fluxcd.io/ssa: "Merge"
 data:
-  ARTIFACT_TAG: "latest-stable"
-  ENVIRONMENT: "production"
-  CLUSTER_NAME: "prod-eu-1"
-  CLUSTER_DOMAIN: "prodeu1.example.com"
+  ARTIFACT_TAG: latest-stable
+  ENVIRONMENT: production
+  CLUSTER_NAME: prod-eu-1
+  CLUSTER_DOMAIN: prodeu1.example.com
+  OCI_SUBJECT_WORKFLOW: release-artifact
+  OCI_SUBJECT_GIT_REF: refs/tags/v\d+\.\d+\.\d+
```

Comment:
If we double-quote this string we will need to add four more back slashes, so I left it without quotes and removed the quotes in the other fields to make it standardized :+1:

Reply:
Please replace `OCI_` with `ARTIFACT_`